src/partials/language/es.html

```diff
@@ -34,15 +34,21 @@
   "meta.source": "Fuente",
   "nav.title": "Navegación",
   "search.config.lang": "es",
+  "search.config.pipeline": "trimmer, stopWordFilter",
```

Comment:
These keys are not necessary, as they're inherited from `en.html`.

Reply:
Done, thanks